collect_app/build.gradle

```diff
@@ -209,22 +209,22 @@ dependencies {
     implementation "com.google.android.gms:play-services-maps:17.0.0"
     implementation "com.google.android.gms:play-services-location:17.0.0"
 
-    implementation "com.google.firebase:firebase-core:17.2.0"
+    implementation "com.google.firebase:firebase-core:17.2.1"
     implementation "com.crashlytics.sdk.android:crashlytics:2.10.1"
 
-    implementation 'com.mapbox.mapboxsdk:mapbox-android-sdk:8.3.0'
+    implementation 'com.mapbox.mapboxsdk:mapbox-android-sdk:8.4.0'
     implementation 'com.mapbox.mapboxsdk:mapbox-android-plugin-annotation-v7:0.6.0'
 
     implementation("com.google.guava:guava:28.1-android")
 
-    implementation("com.google.code.gson:gson:2.8.5") {
+    implementation("com.google.code.gson:gson:2.8.6") {
         exclude group: 'org.apache.httpcomponents'
     }
 
-    implementation("com.google.http-client:google-http-client:1.31.0") {
+    implementation("com.google.http-client:google-http-client:1.32.1") {
```

Comment:
I'm not sure what this is being used for? Just removed it and did a compile and it was all happy. Going to try and a full `checkAll` to see if we can remove it.

Reply:
Yeah looks like we can safely remove this and `com.google.oauth-client:google-oauth-client`

Reply:
I removed them